pkgs/servers/zigbee2mqtt/default.nix

```diff
@@ -1,9 +1,9 @@
-{ pkgs, stdenv, system, dataDir ? "/opt/zigbee2mqtt/data" }:
+{ pkgs, stdenv, system, dataDir ? "/opt/zigbee2mqtt/data", nixosTests }:
```

Comment:
@sweber83 Any particular reason we save state to `/opt/zigbee2mqtt` and not `/var/lib/zigbee2mqtt`?

Reply:
The default dataDir of the package conforms with the documentation of zigbee2mqtt, for non-NixOS nix users.
The NixOS module has its own default, which is `/var/lib/zigbee2mqtt`.
We could change the package to also default to `/var/lib/zigbee2mqtt`, but I think it makes sense to use the upstream default for non-NixOS distros.

Reply:
Thanks for the explanation.